### PR TITLE
Rails 6.1

### PR DESCRIPTION
**Pull request by jherdman**

### CODE REVIEW

File: lib/smart_listing.rb [hunk 1]
```diff
@@ -200,14 +203,25 @@ def set_param key, value, store = @params
       end
     end
 
+    def attribute_method?(klass, attr)
+      klass_name = klass.name.to_s.camelize
+
+      attr = if attr.starts_with?(klass_name) && attr.include?('.')
+               attr.split('.')[1]
+             end
```
Comment (elucid) [inline lib/smart_listing.rb]: nope

**Review by elucid — CHANGES_REQUESTED** [review]

File: lib/smart_listing.rb [hunk 2]
```diff
@@ -200,14 +203,23 @@ def set_param key, value, store = @params
       end
     end
 
+    def attribute_method?(klass, attr)
+      klass_name = klass.name.to_s.camelize
+
+      attr = attr.split('.')[1] if attr.starts_with?(klass_name) && attr.include?('.')
```
Comment (elucid) [inline lib/smart_listing.rb]: How about

```suggestion
      attr = attr.split('.')[1] if attr.starts_with?("#{klass_name}.")
```

**Review by elucid — CHANGES_REQUESTED** [review]
Small suggestion

File: lib/smart_listing.rb (hunk 2 above)
Comment (jherdman) [reply lib/smart_listing.rb]: Looks great!

**Review by elucid — APPROVED** [review]

**Pull request merged**